packages/extension/src/background/wallet.ts

```diff
@@ -75,7 +75,7 @@ export const walletStore = new KeyValueStorage<WalletStorageProps>(
 ) */
 
 export const sessionStore = new ObjectStorage<WalletSession | null>(null, {
-  namespace: "core:wallet:session",
+  namespace: "core:wallet:alphSession",
```

Comment:
Is this shared by Argent-X and our extension wallet?

Reply:
Great catch, it is not, I thought it fixed the ArgentX pop up in some case, but it didn't, reverted it in the next commit. 
Prefixing `ALPH_` for messages fixes the rest of the issues for AgentX popup.